cloud/pkg/common/client/client.go

```diff
@@ -50,8 +47,7 @@ func InitKubeEdgeClient(config *cloudcoreConfig.KubeAPIConfig, enableImpersonati
 	initOnce.Do(func() {
 		kubeConfig, err := clientcmd.BuildConfigFromFlags(config.Master, config.KubeConfig)
 		if err != nil {
-			klog.Errorf("Failed to build config, err: %v", err)
-			os.Exit(1)
+			panic(fmt.Errorf("failed to build kube config, err: %v", err))
```

Comment:
To print the error stack?

Reply:
Yes, and replace `os.Exit(1)` to `panic(err)`. Printing logs and returning the original error does not help us troubleshoot problems. :-)